### PR TITLE
Feature: ApiClass, ApiClassMethod and ApiClassProperty.

### PR DESCRIPTION
**Pull request by MartynasZilinskas**

### CODE REVIEW

File: src/api-helpers.ts [hunk 1]
```diff
@@ -210,4 +222,39 @@ export namespace ApiHelpers {
...
+        }
+
+        return AccessModifier.Public;
+    }
+
+    export function ModifierKindExistsInModifiers(modifiers: ts.NodeArray<ts.Modifier> | undefined, kind: ts.SyntaxKind): boolean {
+        let found = false;
+
+        if (modifiers != null) {
+            modifiers.forEach(modifier => {
```
Comment (GiedriusGrabauskas) [inline src/api-helpers.ts]: Use `.some()`

File: src/definitions/api-class.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        // Members
+        this.members = ApiHelpers.GetItemsFromDeclarationsIds(declaration.members, this.Options);
+
+        // Extends
+        if (declaration.heritageClauses != null) {
+            const extendingList = ApiHelpers.GetHeritageList(declaration.heritageClauses, ts.SyntaxKind.ExtendsKeyword, this.Options);
+
+            if (extendingList.length > 1) {
+                Logger.Log(LogLevel.Warning, "Extending class more than 1 is not supported!");
```
Comment (GiedriusGrabauskas) [inline src/definitions/api-class.ts]: Is it possible?

Comment (MartynasZilinskas) [reply src/definitions/api-class.ts]: In current TypeScript is not possible. Well on second though, when TypeScript will introduce this change we will update then too.

File: src/api-helpers.ts (hunk 1 above)
Comment (MartynasZilinskas) [reply src/api-helpers.ts]: I forgot about it. Thanks 👍

**Pull request merged**